example.php

```diff
@@ -1,7 +1,12 @@
 <?php
 
 $foo = NULL;
+$bar = NULL;
+// some comment
```

Comment:
@CodelingoBotTester rewrite line "// Hello world"

Reply:
@CodelingoBotTester rewrite line "// Hello world"

Reply:
Success! Committed rewritten file.

Reply:
@CodelingoBotTester rewrite line "// Hello world"

Reply:
Success! Committed rewritten file.

Reply:
@CodelingoBotTester rewrite line "// Hello world"

Reply:
Success! Committed rewritten file.

Reply:
@CodelingoBotTester rewrite line "// Hello world"

Reply:
could not get raw file at https://raw.githubusercontent.com/CodelingoBotTester/interactive-rewrite-test/test-branch/example.php

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>